combinetf2/inputdata.py

```diff
@@ -118,13 +118,17 @@ def __init__(self, filename, pseudodata=None, normalize=False):
                     }
 
             self.symmetric_tensor = self.metadata.get("symmetric_tensor", False)
+
+            # TODO this should be deprecated in favor of systematic_type normal
```

Comment:
Can you remove the exponential_transform functionality in this PR?

Reply:
Done